pyscitt/pyscitt/cli/pretty_receipt.py

```diff
@@ -14,16 +14,16 @@
 
 def prettyprint_receipt(receipt_path: Path):
     """
-    Pretty-print a COSE receipt file
+    Pretty-print a receipt which could be a standalone cose file or a transparent statement.
```

Comment:
I think that's fine to do, but then let's change the name from pretty-receipt to pretty-cose, because Statements are not Receipts (they can contain Receipts, but don't have to).

Reply:
Some people think receipt is a transparent statement and some that it is a standalone receipt. This is effectively to make sure it prints even if user provides a transparent statement by accident